src/libs/OptionsListUtils.ts

```diff
@@ -535,6 +540,12 @@ function getLastMessageTextForReport(report: OnyxEntry<Report>): string {
         lastMessageTextFromReport = lastReportAction?.message?.[0].text ?? '';
     } else if (ReportActionUtils.isCreatedTaskReportAction(lastReportAction)) {
         lastMessageTextFromReport = TaskUtils.getTaskCreatedMessage(lastReportAction);
+    } else if (ReportActionUtils.isMemberChangeAction(lastReportAction)) {
+        const htmlWithoutMutedTags = lodashGet(lastReportAction, 'message[0].html', '').replace(/<[/]?muted-text>/g, '');
+        const htmlWithoutAccountIDBasedMentions = ReportActionUtils.convertAccountIDBasedMentionsToDisplayNames(htmlWithoutMutedTags, allPersonalDetails);
+        lastMessageTextFromReport = htmlWithoutAccountIDBasedMentions.replace(/<a[^>]*>(.*?)<\/a>/gi, '$1');
+    } else {
+        lastMessageTextFromReport = report ? report.lastMessageText || '' : '';
```

Comment:
This is redundant.
```suggestion
```

Reply:
@situchan Why is this redundant? It's a fallback in case the report type we're checking isn't handled above.

Reply:
fallback is already here:

<img width="1156" alt="fallbak" src="https://github.com/Expensify/App/assets/108292595/54bf602b-d718-466c-8e93-53062853b8a3">

Reply:
can we mark this as resolved?